bot_python_sdk/bluetooth_service.py

```diff
@@ -0,0 +1,60 @@
+import time
+import bluetooth
+from bluetooth.ble import DiscoveryService, BeaconService
+from bot_python_sdk.logger import Logger
+
+LOCATION = 'Bluetooth Service'
+
+
+
+class BluetoothService:
+
+    def discover_bluetooth_test(self):
+        try:
+            nearby_devices = bluetooth.discover_devices(lookup_names=True)
+            print("found %d devices" % len(nearby_devices))
+
+            for addr, name in nearby_devices:
+                print("  %s - %s" % (addr, name))
+            discover_response = {"message" : "success"}
+        except Exception as exception:
+            Logger.error(LOCATION, 'Failed discover_bluetooth_test')
+            discover_response = {"message" : "failed"}
+            raise exception        
+        return discover_response
+
+    def discover_ble_test(self):
+        try:
+            service = DiscoveryService()
+            devices = service.discover(2)
+            for address, name in devices.items():
+                print("name: {}, address: {}".format(name, address))
+            discover_response = {"message" : "success"}
+        except Exception as exception:
+            Logger.error(LOCATION, 'Failed discover_ble_test')
+            discover_response = {"message" : "failed"}
+            raise exception
+        return discover_response
+
+
+    def advertising_ble_test(self):
+        try:
+            service = BeaconService()
```

Comment:
I am not sure that BeaconService is the right thing to do.
You can't connect to it and exchange Characteristic to the mobile devices 

Please see:
https://github.com/BankingofThings/BoT-NodeJS-SDK/blob/master/lib/bleno/bleno-service.js